src/actions/sagemaker/test_sagemaker.ts

```diff
@@ -0,0 +1,44 @@
+// import * as chai from "chai"
```

Comment:
Is testing required for MVP?

Reply:
yeah probably. need to get some input from wil or someone at looker about what testing is required here